AdlsSdkSamples/Program.cs

```diff
@@ -18,135 +16,102 @@ class Program
+                stream.Write(writeData, 0, writeData.Length);
+                stream.Seek(0, SeekOrigin.Begin);
+                file.Append(stream, length);
+                file.Flush(length + stream.Length);
             }
 
             // Read the file
-            using (var readStream = client.GetReadStream(fileName))
+            Response<FileDownloadInfo> fileContents = file.Read();
+            using (var readStream =fileContents.Value.Content)
             {
                 byte[] readData = new byte[1024];
 
-                // Seek to offset 24
-                readStream.Seek(24, SeekOrigin.Current);
-
                 // Read 40 bytes at this offset
-                int readBytes = readStream.Read(readData, 0, 40);
+                int readBytes = readStream.Read(readData, 25, 40);
                 Console.WriteLine("Read output of 40 bytes from offset 25: " + Encoding.UTF8.GetString(readData, 0, readBytes));
-
             }
         }
-        private static void PerformConcat(AdlsClient client)
-        {
-            // Create two files
-            string file1 = "/dir1/testConcat1.txt";
-            string file2 = "/dir1/testConcat2.txt";
-            string destFile = "/dir1/testConcat.txt";
-            using (var createStream = new StreamWriter(client.CreateFile(file1, IfExists.Overwrite)))
-            {
-                createStream.WriteLine("This is the first file.");
-            }
-            using (var createStream = new StreamWriter(client.CreateFile(file2, IfExists.Overwrite)))
-            {
-                createStream.WriteLine("This is the second file.");
-            }
 
-            // Concatenate the files
-            List<string> list = new List<string> { file1, file2 };
-            client.ConcatenateFiles(destFile, list);
-
-            // Print the resultant combined file
-            using (var readStream = new StreamReader(client.GetReadStream(destFile)))
-            {
-                string line;
-                while ((line = readStream.ReadLine()) != null)
-                {
-                    Console.WriteLine(line);
-                }
-            }
-        }
-        private static void TestTokenRefresh(AdlsClient client)
+        private static void TestTokenRefresh(DataLakeFileSystemClient client)
         {
             string path = "/Test/TokenRefresh.txt";
-
+            DataLakeFileClient file = client.GetFileClient(path);
             // Create file
-            using (var ostreamWriter = new StreamWriter(client.CreateFile(path, IfExists.Overwrite, "")))
+            using (var stream = new MemoryStream(Encoding.UTF8.GetBytes("This is the first file.")))
             {
-                ostreamWriter.WriteLine("This is first line.");
+                file.Upload(stream, true);
             }
 
             // Wait 65 mins
             Console.WriteLine("Wait for 65 minutes to illustrate token refresh");
             Thread.Sleep(65 * 60 * 1000);
```

Comment:
So this is pretty subpar to have in a sample...  If I ran this blindly I'd kill the program but then be annoyed it didn't clean up the resources it created.  I don't think we need to demo this explicitly.  It's fine to just say it refreshes.

Reply:
@tg-msft Updated.